libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -130,6 +131,20 @@ bool AP_ESC_Telem::get_temperature(uint8_t esc_index, int16_t& temp) const
     return true;
 }
 
+// get the highest ESC temperature in centi-degrees if available, returns true on success
+bool AP_ESC_Telem::get_highest_temperature(int16_t& temp) const
+{
+    uint8_t valid_escs = 0;
+
+    for (uint8_t i = 0; i < ESC_TELEM_MAX_ESCS; i++) {
+        if (get_temperature(i, temp)) {
```

Comment:
In what way is that getting the highest temperature?

Reply:
You're right it is not, something has been lost in translation. Fixed